### PR TITLE
Korean GP names update - with fixes

### PR DESCRIPTION
**Pull request by mechu1205**
Small letters after hyphens. e.g. Ok-Gyun -> Ok-gyun
GyeBaek -> Gyebaek.

### CODE REVIEW

**Comment by mechu1205** [comment]
Closed the first PR because I made a new branch with the fix.
Sorry if this causes confusion, I'm not too familiar with Github.

File: Assets/Python/GreatPeople.py [hunk 1]
```diff
@@ -1148,88 +1148,110 @@ def setup():
...
 		"Yi Hwang", # 16th
 		"Yi I", # 16th
+		"Jo Shik", # 16th
+		"Song Si-yeol", # 17th
 		"Yi Seung-hun", # 18th
 		iIndustrial,
 		"Choe Je-u", # 19th
 		iGlobal,
-		"Sun Myung Moon", # 20th
+		"Moon Sun-Myung", # 20th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Lower case.

File: Assets/Python/GreatPeople.py [hunk 2]
```diff
@@ -1148,88 +1148,110 @@ def setup():
...
+		"Kim Su-Hwan", #20th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Lower case.

File: Assets/Python/GreatPeople.py [hunk 3]
```diff
@@ -1148,88 +1148,110 @@ def setup():
...
 		"Jeong Dojeon", # 14th
+		"Yi Saek", # 14th
+		"Jeong Mongju", # 14th
 		iRenaissance,
-		"Yi Hwang", # 16th
-		"Yi Ji-ham", # 16th
+		"Hwang Hui", # 15th
+		"Ryu Seong-ryong", # 16th
+		"Yi Won-ik", # 16th
+		"Che Je-Gong", # 18th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Lower case.

**Review by dguenms — CHANGES_REQUESTED** [review]

**Comment by mechu1205** [comment]
Fixed.

**Review by dguenms — APPROVED** [review]

**Pull request merged**